src/components/HeaderForm.js

```diff
@@ -21,46 +21,46 @@ const dataTypes = {
   MOUNTED_FILES: "mounted files",
   EXAMPLES: "examples",
 };
+const EMPTY_STATE = {
+  xgSelectOptions: ["none"],
+  xgSelect: "none",
+
+  gbwtSelectOptions: ["none"],
+  gbwtSelect: "none",
+
+  gamSelectOptions: ["none"],
+  gamSelect: "none",
+
+  bedSelectOptions: ["none"],
+  bedSelect: "none",
+
+  regionSelectOptions: ["none"],
+  // This tracks several arrays of BED region data, stored by data type, with
+  // one entry in each array per region.
+  regionInfo: {},
+  regionSelect: "none",
+
+  pathSelectOptions: ["none"],
+  pathSelect: "none",
+
+  xgFile: undefined,
```

Comment:
At some point we should add some comments here avout what all these fields mean and why we have e.g. both an `xgSelect` and an `xgFile`.